scheduler/replica_scheduler.go

```diff
@@ -678,14 +677,14 @@ func (rcs *ReplicaScheduler) GetDiskSchedulingInfo(disk longhorn.DiskSpec, diskS
 	return info, nil
 }
 
-func (rcs *ReplicaScheduler) CheckReplicasSizeExpansion(v *longhorn.Volume, oldSize, newSize int64) (err error) {
+func (rcs *ReplicaScheduler) CheckReplicasSizeExpansion(v *longhorn.Volume, oldSize, newSize int64) (diskScheduleMultiError util.MultiError, err error) {
```

Comment:
NIT: returning 2 errors looks a little bit weird

Reply:
Yeah, I'm thinking how to refactor the `MultiError, error` in replica scheduler as well as this check function.

Reply:
Sure. We can check it later. it's not a high-priority issue.